scalableminds/webknossos
# Add brush fill feature

Using the brush tool, any enclosed areas that are painted in one stroke will be filled automatically.

I also updated the default zoom value when opening a volume tracing from 0.1 to 1.0 :tada:
Although that is a tracingstore change, I think it's ok to deploy it together with the next bigger tracingstore changes.

![brush-fill-4](https://user-images.githubusercontent.com/1702075/51846983-02d34480-231b-11e9-86f2-2d8c4b0c9bd0.gif)

### URL of deployed dev instance (used for testing):
- https://brushfill.webknossos.xyz

### Steps to test:
- Open a volume tracing and use the brush tool. Paint a circle - the inside should be filled automatically.
- Paint an open circle, nothing should be filled automatically.
- Paint an "8" or anything else - the enclosed areas should be filled automatically.
- The trace tool should still work as expected.

### Issues:
- fixes #3488 

------
